test/morgan.js

```diff
@@ -822,6 +822,79 @@ describe('morgan()', function () {
         test.write('0')
       })
     })
+
+    describe('a custom', function () {
+      function token (req, res, arg) {
+        return arg
```

Comment:
the other tests for tokens also don't confirm this, unclear on intent of adding those tests to this specific one.